integration-test/src/main/java/uk/gov/gchq/gaffer/integration/AbstractStoreITs.java

```diff
@@ -46,8 +46,6 @@ public AbstractStoreITs(final StoreProperties storeProperties, final Schema sche
         this.skipTestMethods = AbstractStoreIT.getSkipTestMethods();
         AbstractStoreIT.setStoreSchema(schema);
         AbstractStoreIT.setStoreProperties(storeProperties);
-        System.out.println(storeProperties);
-        System.out.println(schema);
```

Comment:
I noticed these when I was restructuring this class a few months ago. I wasn't sure what they were for so left them in. There doesn't seem to be any purpose for these statements, so if there's no other objection I don't see why we shouldn't remove them.